src/core/load-fragment.c

```diff
@@ -2744,6 +2744,13 @@ int config_parse_environ(
                 return 0;
         }
 
+        const Specifier table[] = {
+                COMMON_SYSTEM_SPECIFIERS,
+                COMMON_TMP_SPECIFIERS,
+                COMMON_CREDS_SPECIFIERS(LOOKUP_SCOPE_USER),
```

Comment:
The conf parser is also used by parsing system.conf, so the scope needs to be conditionalized.

Reply:
Please see if the new version is correct.